Lib/locale.py

```diff
@@ -185,8 +185,14 @@ def _format(percent, value, grouping=False, monetary=False, *additional):
         formatted = percent % ((value,) + additional)
     else:
         formatted = percent % value
+    if percent[-1] in 'eEfFgGdiu':
+        formatted = _localize(formatted, grouping, monetary)
+    return formatted
+
+def _localize(formatted, grouping=False, monetary=False):
+    # Transform formatted as locale number according to the locale settings
```

Comment:
Can you enclose the comment around triple double quotes so it becomes a docstring (https://www.python.org/dev/peps/pep-0257/)?

Reply:
It is kind of pointless to make docstring for private method.

Reply:
Yeah, you're right, it doesn't matter here whether we have a docstring.